src/WebdriverClassicDriver.php

```diff
@@ -77,7 +83,10 @@ class WebdriverClassicDriver extends CoreDriver
 
     private const W3C_WINDOW_HANDLE_PREFIX = 'w3cwh:';
 
-    private ?RemoteWebDriver $webDriver = null;
+    /**
+     * @var TWebDriver|null
```

Comment:
anything using phpstan type aliases should use a `phpstan-` prefix for `@var`, `@param` and `@return` to avoid issues with tools reading phpdoc without knowing about phpstan type aliases, which would treat this `TWebDriver` as a class name